pkgs/development/python-modules/glyphtools/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, beziers
+, glyphslib
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "glyphtools";
+  version = "0.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-PXwXHWMJbsi6ZtN+daaXAnlw3gV5DFAhyRxdBa7UP+M=";
+  };
+
+  propagatedBuildInputs = [
+    beziers
+    glyphslib
+    numpy
+  ];
```

Comment:
Nit: Usually in Python's `.pkgs`, explicitly enable or disable tests with `doCheck`. Could you verify that indeed tests are being run here (with the default `doCheck = true` value)?

Reply:
Yes, I got the tests passing on all these packages (except ones where I couldn't find any reasonable test suite to run). I'll add an explicit `doCheck = true` to make it clear there are tests.